src/main/twirl/gitbucket/core/pulls/mergeguide.scala.html

```diff
@@ -199,6 +222,15 @@
 </div>
 
 <script>
+
+$(function(){
+  $('#ready-for-review').click(function(){
+    $.post('@helpers.url(originRepository)/pull/@issue.issueId/update_draft', function(data, status){
+      location.reload();
+    })
+  });
+});
```

Comment:
If you want to submit a request by form, you don't need to send a request by Ajax. In opposite, if you want to send a request by Ajax, you don't need to define `<form>` and form submission related attributes like `validate` and `formaction` in `<button>`.

Reply:
ok, will change acccordingly..